Module that provides add_integer function
======================
Prototype: def add_integer(a, b=98):
-------------------

First import function, (first line) then test:

>>> add_integer = __import__('0-add_integer').add_integer

>>> add_integer(3, 8)
11
>>> add_integer(-3, 8)
5
>>> add_integer(3, -8)
-5
>>> add_integer(0)
98
>>> add_integer(1 * 3)
101
>>> add_integer(1 % 3)
99
>>> add_integer(1 + 3)
102
>>> add_integer('a', 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/holbertonschool-higher_level_programming/python-test_driven_development/0-add_integer.py", line 12, in add_integer
    raise TypeError('a must be an integer')
TypeError: a must be an integer
>>> add_integer(3, 'a')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/holbertonschool-higher_level_programming/python-test_driven_development/0-add_integer.py", line 14, in add_integer
    raise TypeError('b must be an integer')
TypeError: b must be an integer
>>> add_integer(1, 5, 10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: add_integer() takes from 1 to 2 positional arguments but 3 were given
>>> a = float('int')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: could not convert string to float: 'int'
>>> a = float('nan')
>>> b = 9
>>> add_integer(a, b)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/holbertonschool-higher_level_programming/python-test_driven_development/0-add_integer.py", line 12, in add_integer
    raise TypeError('a must be an integer')
TypeError: a must be an integer
>>> a = float('Nan')
>>> b = 9
>>> add_integer(a, b)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/holbertonschool-higher_level_programming/python-test_driven_development/0-add_integer.py", line 12, in add_integer
    raise TypeError('a must be an integer')
TypeError: a must be an integer
>>> a = float("inf")
>>> b = 12
>>> add_integer(a, b)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/holbertonschool-higher_level_programming/python-test_driven_development/0-add_integer.py", line 12, in add_integer
    raise TypeError('a must be an integer')
TypeError: a must be an integer